File: backend/push-service/src/main/java/com/example/pushservice/fcm/dto/FcmMessageDto.java
package com.example.pushservice.fcm.dto;

import lombok.Data;
import lombok.RequiredArgsConstructor;

import java.util.Map;

@Data
@RequiredArgsConstructor
public class FcmMessageDto {
    private String topic;
    private int type;
    private String title;
    private String body;
    private String image;
    private Map<String, String> data;
}

File: backend/push-service/src/main/kotlin/com/example/pushservice/fcm/dto/FcmMessageDto.kt
package com.example.pushservice.fcm.dto

/** ws-service 가 방(topic) 전체에 보내는 채팅 푸시. */
data class FcmMessageDto(
    var topic: String? = null,
    var type: Int = 0,
    var title: String? = null,
    var body: String? = null,
    var image: String? = null,
    var data: Map<String, String>? = null,
)
